README.md

```diff
@@ -1778,6 +1778,35 @@ Cromwell also accepts two [workflow option](#workflow-options) related to call c
 
 > **Note:** If call caching is disabled, the to workflow options `read_from_cache` and `write_to_cache` will be ignored and the options will be treated as though they were 'false'.
 
+## Docker Tags
```

Comment:
@katevoss @vdauwera In terms of the tutorials and other best practice WDL stuff (vs GATK best practices) we need to make a big deal about this